doc/userguide/rules/payload-keywords.rst

```diff
@@ -691,11 +693,25 @@ the reassembled stream.
 The checksums will be recalculated by Suricata and changed after the
 replace keyword is being used.
 
+.. _pkt-data:
+
+pkt_data
+--------
+
+``pkt_data`` is a sticky buffer that allows for payload inspection from the
+beginning of the normalized packet data. As a sticky buffer, ``pkt_data``
+requires the usage of the :ref:`content` keyword for the buffer inspection.
+
+Example of ``pkt_data`` in a signature:
+
+.. container:: example-rule
+
+   alert tftp any any -> any any (msg:"TFTP Test Rule"; :example-rule-emphasis:`pkt_data; content:"rfc1350";` sid:1; rev:1;)
```

Comment:
this rule is the same if `pkt_data` would not be specified

The only real use of pkt_data is to "reset" a previous sticky buffer, e.g.
`http.header; content:"header data"; pkt_data; content:"packet/stream data";`

Reply:
I see now, sorry for the noise with this PR, I'll try a better explanation. Thanks!!